### DISCUSSION TITLE
Can github access content in private repositories?

### DISCUSSION THREAD

**Discussion by DejanKostyszyn** [community/community#22314] [Other Feature Feedback, Questions, & Ideas]
<p>Hello all,</p>
<p>can owners and/or developers of github access the content of users’ private repositories, e.g. reading it? What is the state of data privacy?</p>
<p>Greetings,</p>
<p>DK</p>

**Comment by RDIL** [comment]
<p>Not unless you specifically consent to them viewing the content. For example one time I needed support on a private repo, they needed me to approve their request to actually be granted temporary access. Just don’t do anything you wouldn’t want law enforcement to see :wink:</p>

**Comment by CleanCut** [comment, marked as answer]
<p>That’s a good summary. There are a lot of additional details on the <a href="https://help.github.com/articles/github-privacy-statement/" rel="nofollow noopener">GitHub Privacy Statement</a> page.</p>
